app/containers/Scatter/actions.js

```diff
@@ -61,15 +61,16 @@ export function detachedAccount() {
   };
 }
 
-export function refreshAccountData() {
```

Comment:
I think there was an issue with the rebasing. We want to keep this one.

Reply:
Sorted.